### PR TITLE
[T19] Create API route to create a FEEDBACK

### PR DESCRIPTION
**Pull request by absirgue**
Dear beautiful people,

Ticket is done and all behaviors detailed are implemented.

Important note. It is made so that if a classID that's incorrect is submitted by the feedback (or any other) form it will give a status of 412 which the bot will be able to catch to notify the user if this specific error occurred.

Thanks a ton!

Anton
fox

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -124,9 +125,65 @@ app.post("/rescheduling_request", function (req, res) {
   console.log("Request Received: POST a reschedulling request");
 });
 
-// Empty route to POST feedback requests.
+/**
+ * Creates a new Feedback record.
+ *
+ * The POST request to this endpoint should hold 2 parameters:
+ * class_ID: the ID of the class that is requested to be cancelled
```
Comment (Ali-Alkhars) [inline app.js:132]: This comment needs to be edited

File: queries/feedbacks.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        function (err, recordset) {
+          if (err) {
+            console.log(err);
+            res.status(400).json({ error: err });
+            return null;
+          } else {
+            if (Object.values(recordset.recordset[0])[0] === 0) {
+              callback();
+            } else {
+              res.status(400).json({
```
Comment (Ali-Alkhars) [inline queries/feedbacks.js:62]: Could you give this a unique ID, like 410 so I could use it to notify the users?

File: app.js (hunk 1 above)
Comment (Ali-Alkhars) [reply app.js:142]: @absirgue You still didn't change this. Check the class_ID comment, it shouldn't be "cancelled"

**Review by Ali-Alkhars — APPROVED** [review]
Amazing work Anton! I will approve this but please change the comment which I pointed out.

**Review by TunaAtasayar — APPROVED** [review]
very well done Anton

**Pull request merged**